src/test/scala/uk/gov/hmrc/hmrcfrontend/views/config/HmrcPageHeadingSpec.scala

```diff
@@ -0,0 +1,309 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.views.config
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.i18n.{DefaultLangs, Lang, Messages}
+import play.api.test.Helpers.stubMessagesApi
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.HtmlContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.fieldset.Legend
+import uk.gov.hmrc.govukfrontend.views.viewmodels.label.Label
+
+class HmrcPageHeadingSpec extends AnyWordSpec with Matchers {
+
+  implicit val messages: Messages =
+    stubMessagesApi(
+      Map(
+        "en" -> Map("hmrcPageHeading.visuallyHiddenSectionPrefix" -> "This section is (english translation)")
+      ),
+      new DefaultLangs(Seq(Lang("en")))
+    ).preferred(Seq(Lang("en")))
+
+  "HmrcPageHeadingLabel" must {
+    "construct expected label view model" in {
+      HmrcPageHeadingLabel(text = "What is your name?", section = "Personal details") mustBe Label(
+        isPageHeading = true,
+        content = HtmlContent(
+          """What is your name? """
+            + """<span class="govuk-caption-xl hmrc-caption-xl">"""
+            + """<span class="govuk-visually-hidden">This section is (english translation) </span>"""
```

Comment:
Relating to comment above, I would expect the visually hidden text to match what I passed to it.